gems/pending/spec/support/test_env_helper.rb

```diff
@@ -0,0 +1,177 @@
+require 'yaml'
+require 'fileutils'
+require 'pathname'
+require 'vcr'
+
+GEMS_PENDING_ROOT ||= File.expand_path(File.join(__dir__, "../.."))
+
+class TestEnvHelper
+  attr_reader :relative_dir, :absolute_recording_dir, :config_file
+
+  SPEC_DIR       = File.join(GEMS_PENDING_ROOT, 'spec')
+  SPEC_PATH      = Pathname.new(SPEC_DIR)
```

Comment:
I originally only added `SPEC_PATH` to optimize some of the calls to `relative_path_from`.
But you're right, it would be cleaner using `Pathname` to represent all the paths.